omisego-sdk/src/main/java/co/omisego/omisego/websocket/enum/SocketEvent.kt

```diff
@@ -0,0 +1,56 @@
+package co.omisego.omisego.websocket.enum
+
+/*
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 3/5/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+import co.omisego.omisego.constant.enums.OMGEnum
+
+enum class SocketEventSend(override val value: String) : OMGEnum {
+
+    /**
+     * event used to keep the connection open.
+     */
+    HEARTBEAT("heartbeat"),
+    /**
+     * event used to join a channel.
+     */
+    JOIN("phx_join"),
+
+    /**
+     * event used to leave a channel.
+     */
+    LEAVE("phx_leave");
+
+    override fun toString(): String = this.value
```

Comment:
Do you need to override this everytime? Can't you just do this in `OMGEnum` once? 🤔

Reply:
The interface can't implement method of `Any` class

Reply:
Oh my bad, didn't remember it was an interface 😄